control-plane/agents/src/bin/core/volume/scheduling.rs

```diff
@@ -126,3 +126,20 @@ pub(crate) async fn target_node_candidate(
         Some(node) => Ok(node.clone()),
     }
 }
+
+/// List, filter and collect replicas for which the pools have the required capacity
+/// for resizing the replica.
+pub(crate) async fn pool_capacity_for_repl_resize(
```

Comment:
naming needs changing now I guess?
```suggestion
pub(crate) async fn resizeable_replicas(
```